invenio_circulation/transitions/transitions.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
+# Copyright (C) 2018 RERO.
+#
+# Invenio-Circulation is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio Circulation custom transitions."""
+
+from invenio_circulation.errors import TransitionValidationFailed
+from invenio_circulation.transitions.base import CallableTransition, \
+    Transition, is_pickup_at_same_library, should_item_be_returned
+
+
+class PendingToItemAtDesk(Transition):
```

Comment:
We could maybe call the transition with the action names as in the states chart, for example `ValidateForDesk`/`ValidateForTransit` (For/To)